frontend/packages/ceph-storage-plugin/integration-tests/mocks/storage-class.ts

```diff
@@ -1,10 +1,16 @@
-export const storageClass = {
+export const mockNoProvisionerSC = {
   apiVersion: 'storage.k8s.io/v1',
   kind: 'StorageClass',
-  metadata: {
-    name: 'test-sc',
-    namespace: 'openshift-storage',
-  },
+  metadata: { name: 'mock-no-prov-sc' },
   provisioner: 'kubernetes.io/no-provisioner',
   reclaimPolicy: 'Delete',
 };
+
+export const mockSC = {
+  apiVersion: 'storage.k8s.io/v1',
+  kind: 'StorageClass',
+  metadata: { name: 'mock-ebs-sc' },
```

Comment:
```suggestion
  metadata: { name: 'ebs-sc' },
```

Reply:
Lets have `test-ebs-sc` to differentiate it as a mock stoarge class for testing pupose